scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -85,6 +85,11 @@
                      'Optional (defaults to docker). Prefix added to the start of all docker logs. ',
                      convert_to=str, default='docker')
 
+define_config_option( __monitor__, 'docker_short_lived_interval',
```

Comment:
I think we should default for this to be on.  I'm not as concerned about existing customers seeing more containers show up then they used to having... it's probably not common folks have short lived containers.  I'm more concerned about people not getting all of this containers.

Reply:
No problem.